CRNotifications/Classes/CRNotification.swift

```diff
@@ -34,38 +34,42 @@ public class CRNotificationView: UIView, CRNotification {
         label.translatesAutoresizingMaskIntoConstraints = false
         label.font = UIFont.systemFont(ofSize: 13, weight: .semibold)
         label.textColor = .white
-        label.numberOfLines = 2
+        label.numberOfLines = 0
         return label
 	}()
+    
+    private lazy var labelsContainerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 0.0
+        return stackView
+    }()
+    
 	
     private var completion: () -> () = {}
-    private var type: CRNotificationType
-    public var delegate: CRNotificationDelegate?
+    public var onClickDelegate: CRNotificationDelegate?
```

Comment:
Why is the delegate property renamed? There is no point in limiting the delegate property to such a narrow purpose. When you need other delegate methods you will need new specific protocols. It just polutes the codebase.